FoodBookApp/UI/Views/CreateReview/CreateReview2View.swift

```diff
@@ -133,6 +133,7 @@ struct CreateReview2View: View {
                                             let reviewId = try await model.addReview(review: newReview)
                                             try await model.addReviewToSpot(spotId: spotId, reviewId: reviewId)
                                             if (DBManager().draftExists(spot: spotId) && draftMode) {
+                                                DBManager().deleteImage(spot: spotId)
```

Comment:
Once the review is uploaded, it deletes the saved image (if found) from the storage